tests/main/microk8s-smoke/task.yaml

```diff
@@ -17,6 +17,7 @@ systems:
   - -fedora-42-*         # fails to start service daemon-containerd
   - -ubuntu-14.04-*      # doesn't have libseccomp >= 2.4
   - -arch-linux-*        # XXX: no curl to the pod for unknown reasons
+  - -opensuse-*          # install hook fails
```

Comment:
How does it fail? Is it our fault? Should we file a tracking bug?

Reply:
it's the same problem as on fedora/centos